Core/Core/CourseUpgrade/View/UpgradeCourseView.swift

```diff
@@ -11,8 +11,8 @@ import Swinject
 public enum CourseAccessErrorHelperType {
     case isEndDateOld(date: Date)
     case startDateError(date: Date?)
-    case auditExpired(date: Date?, sku: String, courseID: String, pacing: String, screen: CourseUpgradeScreen)
-    case upgradeable(date: Date?, sku: String, courseID: String, pacing: String, screen: CourseUpgradeScreen)
+    case auditExpired(date: Date?, sku: String, courseID: String, pacing: String, screen: CourseUpgradeScreen, lmsPrice: Double)
+    case upgradeable(date: Date?, sku: String, courseID: String, pacing: String, screen: CourseUpgradeScreen, lmsPrice: Double)
```

Comment:
How about breaking these cases to multilines to silence the `Line Length Violation`  warning?

Reply:
done